doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst

```diff
@@ -37,14 +37,14 @@ In such case, this minimal implementation includes only a call to the default si
 
 With this call, the device will be able to join the Zigbee network or join a new network when it leaves the previous one.
 
-In general, using the default signal handler is worth considering because of the following reasons:
+In general, using the default signal handler is worth considering because:
```

Comment:
We should have a full sentence here, so I prefer to leave as is.

Reply:
I don't fully agree, but I changed this back as it was.